modules/darwin/common/default.nix

```diff
@@ -32,6 +32,17 @@
     HostKey /etc/ssh/ssh_host_ed25519_key
   '';
 
+  launchd.daemons.nix-build-cleanup = {
+    script = "${pkgs.findutils}/bin/find /nix/var/nix/builds -delete || true";
```

Comment:
This could take a creation time flag to only run the cleanup for old build dirs.

Reply:
This also runs on boot?